file-formats/desd/desd-v1.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/desd/desd-v1.json",
+  "title": "CDS Logical External Schema",
+  "description": "CDS logical external schema",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "Format Version",
+      "description": "Format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
```

Comment:
Can you please regenerate the schema using programm saff_generate_repo in your development system. We did a change in the programm some weeks ago and "pattern": "^[a-z]+$" and "maxLength": 2, should no longer be in the schema

Reply:
Hi Nicolas, thanks for the review. The new schema is added.